app/components/contact-page-components/SuccessScreen.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
```

Comment:
I know you mentioned you're getting errors when you remove this import. To address this, try the following steps:
- npm install typescript@latest (This ensures you use TypeScript version 4.1 or above.)
- update your tsconfig.json to set `"jsx": "react-jsx"`

Next.js has built-in TypeScript support, so once these updates/changes are reflected, removing import React from "react" should not cause any issues.